CalculatorMVVM/CalculatorMVVM/ViewModels/CalculatorViewModel.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using Xamarin.Forms;
+using System.Threading.Tasks;
+using System.Linq;
+
+namespace CalculatorMVVM.ViewModels
+{
+    public class CalculatorViewModel : ViewModelBase
+    {
+        public CalculatorViewModel() { }
+
+        private Command _clickCommand;
+        private string _sum;
+        private string _amount;
+        private string _totalAmount;
+        const int _persenteg = 17;
```

Comment:
Constant field must be at the top of fields list and should be named by camel style approach